2.4/examples/replica/mongo_controller.json

```diff
@@ -1,134 +1,224 @@
 {
-  "kind": "Config",
-  "apiVersion": "v1beta1",
-  "id": "mongo-example",
-  "items":[
+  "kind": "Template",
+  "apiVersion": "v1",
+
+  "metadata": {
+    "name": "mongo-replica-example",
+    "creationTimestamp": null,
+    "annotations": {
+      "description": "MySQL Replication Example",
+      "icon": "icon-database",
+      "tags": "database,mongodb,replication"
+    }
+  },
+  "parameters": [
     {
-      "kind": "Service",
-      "apiVersion": "v1beta2",
-      "id": "mongodb",
-      "portalIP": "None",
-      "labels": {
-        "name": "mongodb"
-      },
-      "port": 27017,
-      "selector": {
-        "name": "mongo-replica"
-      }
+      "name": "MONGODB_USER",
+      "description": "The username for MongoDB account to be created",
+      "value": "user"
     },
     {
-      "kind": "Secret",
-      "apiVersion": "v1beta3",
+      "name": "MONGODB_PASSWORD",
+      "description": "Password for the MongoDB user account",
+      "generate": "expression",
+      "from": "[a-zA-Z0-9]{12}"
+    },
+    {
+      "name": "MONGODB_DATABASE",
+      "description": "Password for the MongoDB user account",
+      "value": "userdb"
+    },
+    {
+      "name": "MONGODB_ADMIN_PASSWORD",
+      "description": "Password for the MongoDB admin user",
+      "generate": "expression",
+      "from": "[a-zA-Z0-9]{12}"
+    },
+    {
+      "name": "MONGODB_REPLICA_NAME",
+      "description": " The name of the replSet ",
+      "value": "rs0"
+    },
+    {
+      "name": "MONGODB_SERVICE_NAME",
+      "description": "The name of the MongoDB Service (used to DNS lookup, default: 'mongodb')",
+      "value": "mongodb"
+    },
+    {
+      "name": "MONGODB_KEYFILE_VALUE",
+      "description": "The value of the MongoDB Key",
+      "value": "WGjNxLzJUVkEwd25ualhJNmxXODl0NlBFdGw4YmM1MENTTUFGaU9ncDh3U3JCWTA2YVZGamdBdHlYQwpoQlhNUXNQVmUxcFRYYjRPTnRnVE52d0prckVFWmR6Wjd2WWZQclorV3VwK2NsWUlMc3M4QmJuS1g3bjRzbjFJClpjMERKclJZVXNoeEdrcmhQVXNPTlVZM1NDeHorZnFXb1ljcjV1YmE3MDlib3FMQ0N1Vy90Z0laQUhGUkQ0QXYKZ2JZU1owYTQ5NmZMMGx5ZUZxS3R4a0w2aW1OUEM3dExsY3I0OUwzMGdZMkhiQmRHOGhYRnF0TkdPS2pIWVViagoxTEoxNE9nOG0zYWZDYy9DaUY1WGdaZWx2cDFhb3gwZzRGK05pR1lUWVVHTEZNT3k3UjFTc01TRFBIT2NDMWlyCllBN25KZEJoaWl5cTRteVFQa0ZxSmM1TnJEZml1QlA5STFKNEpuQ3dKUml5MThmSzNoNjJwMzBkeWNWK2dkMkQKRTk3Y0N3NzdRSmtTdXRsY29oMWZwQ2gwSVlhdDNNalBYSCtacDRkdzZFZ3Q3STlYRmJQeHdqaGk1NW5icXhodwpYbVl1R1RETDVBUm55OFNZWUdVa0dXWk5VZzRsUlhSV0xGSTdCTGdnQmJ3QjhCQzl2eS90YmJ6a0xoVjZRUnJlCm1XZVYxRVhDMDYwM3FqT1cvcEFkN2JodFkwM3pMYXduUElXZWc2UEpkL3l4T0lYbjFsN1pHdkxpRzhsbXlpY3EKeVZyS1c0M3V2Nk1tTVZWam5BL1B2MituclBuSWcyQVgrdVJWc1JVdFRQZXpta0FQOEJCNjhaRUJscWhMUXNpMQpQSVhuUFdHMVozN3JYVjVQZHlyMTJrWUlhQzBBM0NRNDNMakRjYjJTT3FNNnlmOEdzc2hqamZnYXpiZEcraEw3Ck9PMU9LeVhyWEFWamx4dGRydm9UQ0UwcEFYQjFJSWdqN21NZ2lPM2pNZ0lwSUxSOERqNVdy"
```

Comment:
Was told by @mfojtik that it would be better for the demo purposes to keep it as a value(also he told me about some restriction) but the only restriction I found is taht the string has to be 6-1024 chars long. Will update it, so the OS will generate it